codegen/smithy-ruby-codegen-test/model/component-test/telemetry.smithy

```diff
@@ -0,0 +1,14 @@
+$version: "2.0"
+namespace smithy.ruby.tests
+
+@auth([])
+@http(method: "POST", uri: "/telemetry_test")
+operation TelemetryTest {
```

Comment:
I _could_ potentially remove this and use the `kitchen_sink` instead but I had some auth related failures when I tried. To resolve this, I think I need to pass in some extra config but I figured maybe we want separate components for each (I find it easier to go thru component file vs. looking at kitchen sink).
```
Auth scheme smithy.api#httpApiKeyAuth did not have an identity resolver configured
...
```